packages/eslint-plugin-ckeditor5-rules/lib/rules/require-file-extensions-in-imports.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const { extname, dirname, sep, parse } = require( 'path' );
+const { isBuiltin } = require( 'node:module' );
```

Comment:
Is `node:` prefix required? So far, we do not use prefixes for Node built-in packages. I would stick to the current syntax. Once we migrate imports (not planned yet), we should have a linter rule that accepts only prefixed imports.